submissions/oluwaboriShobayo/DOM/productPage.js

```diff
@@ -0,0 +1,79 @@
+let  Sform = document.getElementById("form");
+     quantity = document.getElementById("quantity")
+     shippingCost = document.getElementById("shippingCost");
+     totalCost = document.getElementById("totalCost");
+     button = document.getElementById("button");
+  let price = document.getElementById("price")
+ 
+
+    function numberValidation (event) {
+    if (quantity.value == '') {
```

Comment:
Kindly do this ` if (quantity.value == '' || !quantity.value.match(numRegex))` inside of writing separate `if` statement for both of them